pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -820,15 +820,31 @@ stdenv.mkDerivation (finalAttrs: {
     };
   };
 
-  meta = with lib; {
+  meta = {
     homepage = "https://www.freedesktop.org/wiki/Software/systemd/";
     description = "A system and service manager for Linux";
-    license = licenses.lgpl21Plus;
-    platforms = platforms.linux;
+    longDescription = ''
+      systemd is a suite of basic building blocks for a Linux system. It
+      provides a system and service manager that runs as PID 1 and starts the
+      rest of the system. systemd provides aggressive parallelization
+      capabilities, uses socket and D-Bus activation for starting services,
+      offers on-demand starting of daemons, keeps track of processes using Linux
+      control groups, maintains mount and automount points, and implements an
+      elaborate transactional dependency-based service control logic. systemd
+      supports SysV and LSB init scripts and works as a replacement for
+      sysvinit. Other parts include a logging daemon, utilities to control basic
+      system configuration like the hostname, date, locale, maintain a list of
+      logged-in users and running containers and virtual machines, system
+      accounts, runtime directories and settings, and daemons to manage simple
+      network configuration, network time synchronization, log forwarding, and
+      name resolution.
+    '';
+    license = with lib.licenses; [ lgpl21Plus ];
```

Comment:
isn't this wrong?
https://github.com/systemd/systemd/commit/5430f7f2bc7330f3088b894166bf3524a067e3d8
mentions MIT bits and GPL2+ bits.

Reply:
Dunno, it was there when I touched it (2).

Reply:
Main site removed MIT: https://github.com/systemd/systemd/blob/main/LICENSE.MIT

Reply:
Check https://github.com/systemd/systemd/blob/720c618397397f958caeb050a1528eb0d6f7a4a6/LICENSES/README.md?plain=1#L37, it explains the state of licensing.

Reply:
Updated.